documentation/src/docs/metrics.md

```diff
@@ -116,204 +134,169 @@ public final class ServiceStarter {
 ```
 
 ## 2.1. Example for Dynamic Counter and OpStatsLogger(Timer)
-This is an example from io.pravega.segmentstore.server.host.handler.PravegaRequestProcessor. In this class, we registered two metrics: One timer (createStreamSegment), one dynamic counter (segmentReadBytes).
+This is an example from io.pravega.segmentstore.server.host.stat.SegmentStatsRecorderImpl.java. In this class, we 
+registered two metrics: One timer (createStreamSegment), one dynamic counter (dynamicLogger).
 ```java
-public class PravegaRequestProcessor extends FailingRequestProcessor implements RequestProcessor {
+public class SegmentStatsRecorderImpl implements SegmentStatsRecorder {
```

Comment:
And this.

Most of this code is irrelevant to metrics.

Reply:
All irrelevant lines removed.